internal/cmd/operator-sdk/bundle/validate/cmd.go

```diff
@@ -23,8 +23,8 @@ import (
 	"github.com/spf13/viper"
 	"k8s.io/apimachinery/pkg/labels"
 
-	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/bundle/validate/internal"
 	"github.com/operator-framework/operator-sdk/internal/flags"
+	"github.com/operator-framework/operator-sdk/internal/util/resultutil"
```

Comment:
The stuff in this package are `cmd/`-related, so can it live in `internal/cmd/util`?